tuf/client/client.go

```diff
@@ -421,8 +419,9 @@ func (c *Client) downloadTargets(role string) error {
 	return nil
 }
 
-func (c *Client) downloadSigned(role string, size int64, expectedSha256 []byte) ([]byte, *data.Signed, error) {
-	raw, err := c.remote.GetMeta(role, size)
+func (c *Client) downloadSigned(role string, size int64, expectedSha256 []byte, consistent bool) ([]byte, *data.Signed, error) {
```

Comment:
I was going to disagree (unexported method, explicit behaviour, blah, blah, blah) with you but in the context of notary, if we have a sha256, we should _always_ do a consistent download so I'll happily make this change :-)